### PR TITLE
change env vars from QD to SK

### PR DESCRIPTION
**Pull request by ajssmith**
Relates to https://github.com/skupperproject/skupper-router/pull/330

### CODE REVIEW

File: client/image_utils.go [hunk 1]
```diff
@@ -1,16 +1,17 @@
...
+	"os"
+
 	"github.com/skupperproject/skupper/api/types"
 	corev1 "k8s.io/api/core/v1"
-	"os"
 )
 
 const (
-	RouterImageEnvKey                 string = "QDROUTERD_IMAGE"
+	RouterImageEnvKey                 string = "SKROUTERD_IMAGE"
```
Comment (grs) [inline client/image_utils.go:11]: I think if we are changing this we should change to SKUPPER_ROUTER_IMAGE.

File: client/image_utils.go [hunk 2]
```diff
@@ -1,16 +1,17 @@
...
 	ServiceControllerImageEnvKey      string = "SKUPPER_SERVICE_CONTROLLER_IMAGE"
 	ConfigSyncImageEnvKey             string = "SKUPPER_CONFIG_SYNC_IMAGE"
-	RouterPullPolicyEnvKey            string = "QDROUTERD_IMAGE_PULL_POLICY"
+	RouterPullPolicyEnvKey            string = "SKROUTERD_IMAGE_PULL_POLICY"
 	ServiceControllerPullPolicyEnvKey string = "SKUPPER_SERVICE_CONTROLLER_IMAGE_PULL_POLICY"
```
Comment (grs) [inline client/image_utils.go:15]: As above I think a better name is SKUPPER_ROUTER_IMAGE_PULL_POLICY.

File: client/router_create.go [hunk 3]
```diff
@@ -484,7 +484,7 @@ func (cli *VanClient) GetRouterSpecFromOpts(options types.SiteConfigSpec, siteId
 	// skupper-router container index
 	// TODO: update after dataplance changes
 	const (
-		qdrouterd = iota
+		skrouterd = iota
```
Comment (grs) [inline client/router_create.go:487]: Maybe just router here?

File: api/types/types.go [hunk 4]
```diff
@@ -51,7 +51,7 @@ const (
 	TransportServiceAccountName   string = "skupper-router"
 	TransportRoleName             string = "skupper-router"
 	TransportRoleBindingName      string = "skupper-router"
-	TransportEnvConfig            string = "QDROUTERD_CONF"
+	TransportEnvConfig            string = "SKROUTERD_CONF"
```
Comment (fgiorgetti) [inline api/types/types.go:54]: Prior to changing this, we need to change the environment variable name in `skupper-router`
as it is still being referred there with the old name.

File: client/router_create.go [hunk 5]
```diff
@@ -621,21 +621,21 @@ func (cli *VanClient) GetRouterSpecFromOpts(options types.SiteConfigSpec, siteId
 			},
 		},
 		})
-		envVars = append(envVars, corev1.EnvVar{Name: "QDROUTERD_AUTO_MESH_DISCOVERY", Value: "QUERY"})
+		envVars = append(envVars, corev1.EnvVar{Name: "SKROUTERD_AUTO_MESH_DISCOVERY", Value: "QUERY"})
```
Comment (fgiorgetti) [inline client/router_create.go:624]: Same for all these environment variables. They must be changed in `skupper-router` first.

File: client/router_update.go [hunk 6]
```diff
@@ -419,11 +419,11 @@ func (cli *VanClient) RouterUpdateVersionInNamespace(ctx context.Context, hup bo
 		updateRouter = true
 	}
 	if renamedSkRouter {
-		// Updating QDROUTERD_CONF env var
-		envQdrouterdConf := kube.GetEnvVarForDeployment(router, "QDROUTERD_CONF")
-		envQdrouterdConf = strings.ReplaceAll(envQdrouterdConf, "qpid-dispatch", "skupper-router")
-		envQdrouterdConf = strings.ReplaceAll(envQdrouterdConf, "qdrouterd", "skrouterd")
-		kube.SetEnvVarForDeployment(router, "QDROUTERD_CONF", envQdrouterdConf)
+		// Updating SKROUTERD_CONF env var
```
Comment (fgiorgetti) [inline client/router_update.go:422]: Actually we need to rename the old QDROUTERD_CONF to SKROUTERD_CONF here.
The same renaming must be done as part of it for all the other renamed variables.

**Comment by ajssmith** [comment]
Defer to a future release.

**Pull request closed without merging**